docs/resiliency-checklist.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Resiliency Checklist
+resourceType: Documentation
+personas: 
+  - Developer
+tags:
+  - openshift
+  - ha
+  - high availability
+  - resilient
+  - devops
+description: How to design your application to have as much uptime as possible.
+---
+
+# Resiliency Checklist
+
+One of the most amazing benefits of working on the Openshift Platform is that, when your application has been designed with a few key ideas in mind, you can avoid many of the regular outages that are almost unavoidable on legacy infrastructure.
+All that time you used to spend with your application down because you needed to patch a server? Gone.
+If a node needs to go down for patching purposes, a correctly designed application can simply spin up another pod on another node, and your users won't even notice.
+
+## What does "Correctly Designed" mean?
+
+In order to take advantage of this feature of the cluster, the development team will have to take a design approach that's a little different from legacy design methods.
+In short, this means that your application needs to be highly available - it should have multiple pods running simultaneously on different nodes. That way, if one pod goes down (because, for example, it was running on a node that is being evacuated for patching), the others - which will be on different nodes - will continue to chug along without issue.
```

Comment:
```suggestion
In short, this means that your application needs to be **highly available** - it should have multiple pods running simultaneously on different nodes. That way, if one pod goes down (because, for example, it was running on a node that is being evacuated for patching), the others - which will be on different nodes - will continue to chug along without issue.
```